Dockerfile

```
@@ -129,4 +130,4 @@ ENTRYPOINT ["/opt/startup.sh"]
 WORKDIR /opt
 
 HEALTHCHECK --interval=1m --timeout=20s --retries=3 \
-  CMD curl --fail $HEALTHCHECK_URL || exit 1
+  CMD if [ $WEBAPP_CONTEXT == "ROOT" ];then (curl --fail ROOT_HEALTHCHECK_URL || exit 1); else (curl --fail HEALTHCHECK_URL || exit 1);fi
```

You're missing the `$` signs for the variables:

```suggestion
  CMD if [ $WEBAPP_CONTEXT == "ROOT" ];then (curl --fail $ROOT_HEALTHCHECK_URL || exit 1); else (curl --fail $HEALTHCHECK_URL || exit 1);fi
```